handler/handler.go

```diff
@@ -375,7 +375,15 @@ func (h *Handler) Run() error {
 				ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
 				if err = h.Execute(ctx, out, opt, h.lastPrefix, h.last, forceBatch); err != nil {
 					lastErr = WrapErr(h.last, err)
-					fmt.Fprintln(stderr, "error:", err)
+					ON_ERROR_STOP, parseErr := strconv.ParseBool(env.All()["ON_ERROR_STOP"])
+					if parseErr != nil {
+						fmt.Fprintln(stderr, "error parsing value for ON_ERROR_STOP:", parseErr)
+					}
+					if ON_ERROR_STOP {
+						return err
```

Comment:
Call `stop()` before returning

Reply:
What would happen if `ON_ERROR_STOP` is enabled in interactive mode?

Reply:
Will add stop() prior to the return.

Good Q! In interactive mode any error causes the CLI to exit, which isn't consistent with current behavior and doesn't seem desirable. Will need to figure out how to get `ON_ERROR_STOP` behavior in interactive mode without causing usql to exit.

Reply:
Ok, both points here should be addressed now.